resources/public/api/conf/3.0/examples/lossClaims/CreateLC/create_request_property_claim.json

```diff
@@ -0,0 +1,6 @@
+{
+  "businessId": "XGIS00000001319",
```

Comment:
Can you update the businessId to XPIS12356589871 as XP is for UK Prop related ids